core/blockchain.go

```diff
@@ -334,15 +360,18 @@ func (bc *BlockChain) loadLastState() error {
 		utils.Logger().Warn().Str("hash", head.Hex()).Msg("Head block missing, resetting chain")
 		return bc.Reset()
 	}
-	// Make sure the state associated with the block is available
-	if _, err := state.New(currentBlock.Root(), bc.stateCache); err != nil {
-		// Dangling block without a state associated, init from scratch
-		utils.Logger().Warn().
-			Str("number", currentBlock.Number().String()).
-			Str("hash", currentBlock.Hash().Hex()).
-			Msg("Head state missing, repairing chain")
-		if err := bc.repair(&currentBlock); err != nil {
-			return err
+
+	if !bc.options.SkipInitialStateValidation {
```

Comment:
Why is this needed? please add comment to explain.

Reply:
Added comment. 
	// Skip state check on initialization.
	// Should be enabled for epoch chain because it doesn't contain valid state.